spring-boot-project/spring-boot-actuator-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -54,6 +54,12 @@
       "description": "Whether to enable liveness and readiness probes.",
       "defaultValue": false
     },
+    {
```

Comment:
This should come before the previous entries as we sort them by alphabetical order.

Reply:
Thanks, I have rearranged it.